includes/multi-currency/MultiCurrency.php

```diff
@@ -455,6 +455,7 @@ private function initialize_available_currencies() {
 	private function initialize_enabled_currencies() {
 		$available_currencies     = $this->get_available_currencies();
```

Comment:
I think you've missed that, the function already has to return an array, or it'll throw an error, it has a return type declaration: https://github.com/Automattic/woocommerce-payments/blob/36b512ad21373783aaeb646f509b66703e19e96b/includes/multi-currency/MultiCurrency.php#L511-L513

Reply:
Right, I think he was referring to checking to make sure it was an array in the method itself so that it always returns an array instead of throwing an error.

Reply:
Oh, yes, that must be it.